README.md

```diff
@@ -1 +1,194 @@
-# Portfolio-Project
\ No newline at end of file
+
+<div align="center">
+ 
+  <img src="#" alt="logo" width="140"  height="auto" />
```

Comment:
- [OPTIONAL] Great job in implementing the changes requested by the previous code Reviewers, However, I kindly suggest that you either add a valid logo link in your readme file or remove it so it will not confuse the readers. :slightly_smiling_face: